packages/frontend/components/authorize/AuthorizeClaim.vue

```diff
@@ -0,0 +1,60 @@
+<script setup lang="ts">
+
+
+
+interface Props {
+  claim: keyof typeof claims,
+}
+
+const props = withDefaults(defineProps<Props>(), {})
+
+const claims = {
+    "identity": {
+        "header": "Identity",
+        "description": "View your basic identity info",
+    },
+    "email": {
+        "header": "Email",
+        "description": "View your email address",
+    },
+    "phone": {
+        "header": "Phone",
+        "description": "View your phone number",
+    },
+    "address": {
+        "header": "Address",
+        "description": "View your address",
+    }
+}
+
+// def header
+const header = computed(() => {
+  try{
+    return claims[props.claim].header
+  }
+  catch{
+    throw new Error("Invalid claim")
+  }
+})
+// def description
+const description = computed(() => {
+  try{
+    return claims[props.claim].description
+  }
+  catch{
+    throw new Error("Invalid claim")
+  }
+})
+
+
+
+</script>
+
+
+<template>
+  <div class="flex flex-col justify-start b-1 b-dark-1 rounded-8px w-100% p-10px">
```

Comment:
You shouldn't use pixels, use the tailwind units (in our case "unocss units") it helps people, who have changed the browser's font settings.

We also want to stick to how classes in tailwind are defined, so `b-1`, `b-dark-1`, `w-100%` etc. would be written incorrectly. (`border`, `border-dark-1`, `width-full` is correct)